src/components/Navbar/Navbar.jsx

```diff
@@ -60,6 +76,25 @@ const Navbar = () => {
               {item.name}
             </NavLink>
           ))}
+
+          {!token ? (
+            <>
+              <p className="divider"></p>
+              {authNavigation.map((item) => (
+                <NavLink
+                  key={item.id}
+                  to={`${item.path}`}
+                  onClick={() => setShow(false)}
```

Comment:
Template literal is redundant over here

Reply:
Fixed - 58c457443ebd34bc9e1ca0b852d1b782157ef834